### PR TITLE
fix: warpify modal

### PR DESCRIPTION
**Pull request by AnnaRybkina**
Styles were migrated from https://github.com/fabric-ds/css/blob/next/src/components/modal.css

Type|Screenshot
---|---
Standard|image
With buttons|image
Must confirm|image image

### CODE REVIEW

**Review by BalbinaK — COMMENTED** [review]
Looks really nice! Just small comments related to deprecated classes :)

File: packages/modal/src/component.tsx [hunk 1]
```diff
@@ -55,27 +55,26 @@ export const Modal = ({
               props.onDismiss();
             }
           }}
-          className={c.modal}
+          className={ccModal.modal}
           tabIndex={-1}
         >
-          <div className={c.title}>
+          <div className={ccModal.title}>
```
[the remark this answers is not in the document]
Comment (AnnaRybkina) [reply packages/modal/src/component.tsx:61]: Nice catch. Added fixes here https://github.com/warp-ds/component-classes/pull/58/files. I tried to understand how `f-grid` could have been used here, https://github.com/fabric-ds/css/blob/next/src/components/grid.css But I'm not sure if something is needed. Do you know?

[the remark this answers is not in the document]
Comment (BalbinaK) [reply packages/modal/src/component.tsx:61]: Hm, looks like it was used to fix some troika vs. fabric issues ([source](https://github.com/fabric-ds/css/blob/01fd80a27461cf0d593911e974853f84963d0681/src/utilities.css#L29-L33)). @Skadefryd do you think we should do something about it in Warp, too?

[the remark this answers is not in the document]
Comment (Skadefryd) [reply packages/modal/src/component.tsx:61]: Well spotted. f-grid should not be needed. Its purpose was when Troika and Fabric ran together on the same page troikas grid was overriding Fabrics grid, the f-grid overided the overide back to fabric setup :)

**Review by Skadefryd — APPROVED** [review]
Looks good to me!

**Pull request merged**